ktor-client/ktor-client-plugins/ktor-client-logging/common/src/io/ktor/client/plugins/logging/Logging.kt

```diff
@@ -239,6 +613,34 @@ public val Logging: ClientPlugin<LoggingConfig> = createClientPlugin("Logging",
     ResponseObserver.install(ResponseObserver.prepare { onResponse(observer) }, client)
 }
 
+private fun Url.pathQuery(): String {
+    return buildString {
```

Comment:
Please consider adding:
-  a non-empty browser test with `window.location`
- test with default request plugin and relative path

Reply:
> test with default request plugin and relative path

Wouldn't I test the `DefaultRequest` plugin then? If not, can you explain the subject under test?

Reply:
Set `window.location.href = "https://ktor.io"`

So this call will give you a different result: https://github.com/ktorio/ktor/blob/c9739da4023dbf3127d5b91bdb621811fc1ee556/ktor-http/jsAndWasmShared/src/io/ktor/http/URLBuilderJs.kt#L9

Reply:
> Set `window.location.href = "https://ktor.io"`
> 
> So this call will give you a different result:
> 
> https://github.com/ktorio/ktor/blob/c9739da4023dbf3127d5b91bdb621811fc1ee556/ktor-http/jsAndWasmShared/src/io/ktor/http/URLBuilderJs.kt#L9

This method is used only with the `URLBuilder.Companion.origin` property, which I don't use at all.